ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs

```diff
@@ -335,9 +334,10 @@ protocolClientInfoShelley =
   Inspection
 -------------------------------------------------------------------------------}
 
+-- TODO: This should be updated as soon as we start preparing for the
+-- hard fork transition out of Shelley.
```

Comment:
We should create a ticket for it with the right label.

Reply:
Created https://github.com/input-output-hk/ouroboros-network/issues/2452 , and informed Volodymyr that this should be part of the "prepare for next hard fork" item.